### ISSUE TITLE
Provide support for mobile devices

### ISSUE THREAD

**Issue by palashbhowmick** [palashbhowmick/mazify-3d#4]
The current version of mazify-3d is not working in mobile device

**Comment by palashbhowmick** [comment]
@Amiralgaby do you want to work on it?

**Comment by Amiralgaby** [comment]
> @Amiralgaby do you want to work on it?

no, my mazify-3d doesn't work too on mobile device.... normally it doesn't come from me (i'm sure)

Not so long ago I had managed to make it work, then as in my game it was rather the system D to achieve features, I had to break something in my project for mobile support.

Now you're telling me that it just happened or that it was always the case?

**Comment by palashbhowmick** [comment]
It was not working in mobile devices from the beginning. I need to update the aframe js in order to make it run in mobile devices

**Comment by Amiralgaby** [comment]
Hi, Excuse me, did it work before on phone ? Is it because of my pull request that it no longer works?

**Comment by Amiralgaby** [comment]
I had the memory to have tried with a recent version, and also the memory to have had something acceptable during a moment on phone...
